glusterd2/volume/volume-utils.go

```diff
@@ -45,3 +45,12 @@ func IsQuotaEnabled(v *Volinfo) bool {
 	}
 	return false
 }
+
+// IsGfproxydEnabled returns true if gfproxyd is enabled for a volume and false otherwise
+func IsGfproxydEnabled(v *Volinfo) bool {
```

Comment:
This should be in the gfproxyd package. Again, plugins shouldn't be modifying core packages.

Reply:
Same here, move this function to `plugins/gfproxyd`